newsroom/photos.py

```diff
@@ -0,0 +1,6 @@
+def set_photo_coverage_href(coverage, slugline):
```

Comment:
imo we should pass there `coverage` and `planning_item`, not the `slugline` which is already kind of aap specific